docs/topics/i18n/formatting.txt

```diff
@@ -60,6 +60,9 @@ to be machine-readable, you will always want unlocalized values. You
 may also want to use localization in selected templates, rather than
 using localization everywhere.
 
+The format used is the one from the settings and under normal circumstances
+the locale dictated format has a higher precedence and will be applied instead.
```

Comment:
Hey @claudep , thanks for your review and response on this, Just for clarity, is it okay to pass a reference to the [globalization settings](https://docs.djangoproject.com/en/5.0/ref/settings/#globalization-i18n-l10n)  as the general formatting settings instead of listing them.

Reply:
It sounds like we still want to do this, it can be a separate commit or another PR 👍